invenio_upgrader/upgrades/invenio_2015_07_14_innodb.py

```diff
@@ -1,65 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
we should create new upgrade recipe that removes this one (see invenio_upgrader_2015_09_23_legacy_removal.py)